### ISSUE TITLE
Calendar looks blank in RTL mode

### ISSUE THREAD

**Issue by fakhergh** [howljs/react-native-calendar-kit#52]
Hi folks,

I'm working on an application that supports i18n, and I tried to implement calendar-kit.
It looks fine on `LTR` but not on `RTL`.

Here is screenshot for the issue:
![Screenshot_1678603907](https://user-images.githubusercontent.com/32802712/224529391-b1b24678-79a2-470e-bdf7-54b696dfccd8.png)

Current used libraries: 

"react-native": "0.70.6"
"react": "18.1.0"
"@shopify/flash-list": "^1.4.1"
"react-native-reanimated": "^2.14.4"
"@howljs/calendar-kit": "^1.13.1"
....

Anyone can help or suggest a solution for me? thanks.

**Comment by fakhergh** [comment]
@howljs

**Comment by fakhergh** [comment]
@howljs Thank you for this amazing package. I really like it

and could you please help me to solve this issue?

**Comment by howljs** [comment]
I’m busy with my work at the company right now ☹️ I’ll try to take the time to write this feature later. Sorry

**Comment by fakhergh** [comment]
Thank you @howljs for your effort.

**Comment by FadiAboMsalam** [comment]
am facing the same issue @howljs @fakhergh  any update on this ?

**Comment by Deblancd** [comment]
For me what fixed it was using direction: 'ltr' inside the Timeline style for content and container
src/components/Timeline/index.tsx

const styles = StyleSheet.create({
  container: { flex: 1, direction: 'ltr' },
  content: { flexGrow: 1, direction: 'ltr' },
});

**Comment by FadiAboMsalam** [comment]
@Deblancd  i believe that will work on iOS but not android

**Comment by derickdecesare** [comment]
Yes I am running into this problem as well on android. It seems to work fine on IOS but the schedule does not render properly with RTL on android. Is there any fix to this other than having the user change their system language setting to something that is RTL?

**Comment by SwikarBhattarai** [comment]
Hi @derickdecesare , @Deblancd, @FadiAboMsalam , did you guys found any solution ?
